package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-trimtrailingspaces",
   "description": "Trim trailing spaces from each line of the selected files",
-  "version": "1.1.0",
```

Comment:
Please do not touch the version numbers, as it is the job of who ever releases the given version